client/src/main/scala/skuber/ext/HorizontalPodAutoscaler.scala

```diff
@@ -41,8 +41,8 @@ object HorizontalPodAutoscaler {
   }
    case class Spec( 
      scaleRef: SubresourceReference,
-     minReplicas: Int = 1,
-     maxReplicas: Int = 1,
```

Comment:
i verified that you are correct, kube kicks it back without `maxReplicas`, i'll change this